src/Repository/TrickRepository.php

```diff
@@ -18,4 +18,17 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, Trick::class);
     }
+
+    /**
+     * @return int|mixed|string
+     */
+    public function findAllRecentWithMedia()
```

Comment:
No needed because you have a getMedias() function inside Trick entity so you can access it directly